mozilla-release/browser/components/customizableui/CustomizableUI.jsm

```diff
@@ -212,7 +212,7 @@ var CustomizableUIInternal = {
 
     let navbarPlacements = [
       "urlbar-container",
-      "search-container",
+      "cliqz-button",
```

Comment:
LGTM, except that I'd leave the original ID commented inside `#if 0 ... #endif` in order to slightly reduce conflict possibility.